src/app.js

```diff
@@ -39,7 +39,7 @@ if (app.get('env') === 'development') {
 
 app.all('*', function (req, res, next) {
   res.setHeader('Access-Control-Allow-Origin', '*')
-  res.header('Access-Control-Allow-Methods', 'POST, PUT, OPTIONS, DELETE, GET')
+  res.header('Access-Control-Allow-Methods', 'POST, PUT, OPTIONS, PATCH, DELETE, GET')
   res.header('Access-Control-Max-Age', '3600')
   res.header(
     'Access-Control-Allow-Headers',
```

Comment:
Please add Access-Control-Allow-Origin to the comma separated values this header takes, since we are using it (-allow-origin) on line 41

Reply:
Access-Control-Allow-Origin added